### PR TITLE
[website_crm_partner_assign] Migrated to new api.

### PR DESCRIPTION
**Pull request by rgo-odoo**
Task: https://www.odoo.com/web#id=12026&view_type=form&model=project.task&action=327
Pad: https://pad.odoo.com/p/r.TkQhEhvIYXjGJx6w

### CODE REVIEW

File: addons/website_crm_partner_assign/views/partner_grade.xml [hunk 1]
```diff
@@ -1,9 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
-<openerp>
-<data>
+<odoo>
 
     <record model="ir.ui.view" id="partner_grade_view">
-        <field name="name">res.partner.grade.website</field>
+        <field name="name">res.partner.grade.inherit</field>
```
Comment (atp-odoo) [inline addons/website_crm_partner_assign/views/partner_grade.xml]: it should be "res.partner.grade.form.inherit"

File: addons/website_crm_partner_assign/__init__.py [hunk 2]
```diff
@@ -1,2 +1,23 @@
...
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
```
Comment (msh-odoo) [inline addons/website_crm_partner_assign/__init__.py]: Use global LICENSE/COPYRIGHT files

File: addons/website_crm_partner_assign/controllers/main.py [hunk 3]
```diff
@@ -1,11 +1,10 @@
 # -*- coding: utf-8 -*-
+
 import werkzeug
-from openerp import SUPERUSER_ID
-from openerp.addons.web import http
-from openerp.addons.web.http import request
+from openerp import http, _
+from openerp.http import request
```
Comment (msh-odoo) [inline addons/website_crm_partner_assign/controllers/main.py]: from odoo...

File: addons/website_crm_partner_assign/controllers/main.py [hunk 4]
```diff
@@ -101,19 +91,13 @@ def partners(self, country=None, grade=None, page=0, **post):
         if country_all:
             url_args['country_all'] = True
 
-        partner_count = partner_obj.search_count(
-            request.cr, SUPERUSER_ID, base_partner_domain,
-            context=request.context)
+        partner_count = PartnerSudo.search_count(base_partner_domain)
```
Comment (msh-odoo) [inline addons/website_crm_partner_assign/controllers/main.py]: You are doing search with same domain on line #100 then do we need this search_count, we should write search here and we can get it using len(partners)

File: addons/website_crm_partner_assign/controllers/main.py [hunk 5]
```diff
@@ -101,19 +91,13 @@ def partners(self, country=None, grade=None, page=0, **post):
...
         pager = request.website.pager(
             url=url, total=partner_count, page=page, step=self._references_per_page, scope=7,
             url_args=url_args)
 
         # search partners matching current search parameters
-        partner_ids = partner_obj.search(
-            request.cr, SUPERUSER_ID, base_partner_domain,
-            order="grade_id DESC",
-            context=request.context)  # todo in trunk: order="grade_id DESC, implemented_count DESC", offset=pager['offset'], limit=self._references_per_page
```
Comment (msh-odoo) [inline addons/website_crm_partner_assign/controllers/main.py]: todo ? check line 101, we can improve it now in this version :)

**Comment by tde-banana-odoo** [comment]
Done in master at https://github.com/odoo/odoo/commit/053982f48d4c824a68ee1e56e897d576530777c7 . Thanks !

**Pull request closed without merging**